person.rb

```diff
@@ -8,9 +8,10 @@ def initialize(age, name = 'Unknown', parent_permission = 'true')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
Getters and Setters for `rentals` should be present, Kindly add the `rentals` to attr_accessor. 👍